unity-ggjj/Assets/Scripts/SaveFiles/AudioSaveDataManager.cs

```diff
@@ -14,10 +14,41 @@ private void Start()
         {
             _settingsSaveData = GetComponent<SettingsSaveDataLoader>().SettingsSaveData;
             
-            _audioGroup.SetFloat("MasterVolume", _settingsSaveData.GameAudioSettings.Master);
-            _audioGroup.SetFloat("MusicVolume", _settingsSaveData.GameAudioSettings.Music);
-            _audioGroup.SetFloat("SFXVolume", _settingsSaveData.GameAudioSettings.Sfx);
-            _audioGroup.SetFloat("DialogueVolume", _settingsSaveData.GameAudioSettings.Dialogue);
+            if (Mathf.Approximately(_settingsSaveData.GameAudioSettings.Master, -1))
```

Comment:
Think i prefer checking if it's less than zero, than the approximately method.
Probably could just do settingsSaveData.GameAudioSettings.Master == -1 too as it's hardcoded as -1.
```suggestion
            if (settingsSaveData.GameAudioSettings.Master < 0))
```

Reply:
Resolved in 2132c6925ee9e12d9341bda2ef49dce36662133e.